product_replenishment_cost_default_seller/models/product_product.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
+# Copyright 2019 PlanetaTIC - Marc Poch <mpoch@planetatic.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, api
+from odoo.models import Model
+
+import odoo.addons.decimal_precision as dp
+
+
+class ProductProduct(Model):
+    _inherit = 'product.product'
+
+    @api.one
```

Comment:
the standard for compute method is to have a funciton named ``_compute_xxx``.
also it's an api.multi. 
api.one is deprecating.

Reply:
Fixed.
Thanks for your feedback